asset_dashboard/models.py

```diff
@@ -248,3 +248,205 @@ class DummyProject(models.Model):
     def __str__(self):
         """"String for representing the model object"""
         return self.name
+
+
+class NaturePreserves(models.Model):
+
+    class Meta:
+        managed = False
+        db_table = '"quercus"."nature_preserves"'
```

Comment:
Needed to quote the schema and table names for this to work. Not sure why, also lost StackOverflow reference, sorry!